docs/content/0.Config/5.Automate/1.Usage.md

```diff
@@ -1,3 +1,44 @@
+---
+title: Usage
+---
+
 # Automate
 
-Not yet documented ...
\ No newline at end of file
+The "Automate" feature can be used to **periodically request one or more PID**s and post the results to an **MQTT server**. Therefore, this feature requires [MQTT configured in WiCAN](../MQTT.md).
+
+Configuration of "Automate" is done on the "Automate" tab.  
+
+## Vehicle Profiles
+The "Automate" feature automatically downloads **vehicle profiles** from GitHub containing some popular PIDs (e.g. SoC, SoH). These vehicle profiles are car vendor and/or model specific. There is a growing [list of supported vehicles](2.Supported_Vehicles.md).
```

Comment:
you have to press the sync button next to the vehicle model drop down list to fetch the vehicle profiles

Reply:
You mean I have to press the sync button initially to get the list? I cannot remember, that I did it ...

Reply:
Yes that's right, it's probably because you had the whole file stored in the device. Once you click the sync you fetch the latest from GitHub

Reply:
fixed